pkg/api/types.go

```diff
@@ -504,20 +506,32 @@ func (l *VolumeList) Set(value string) error {
 	if len(value) == 0 {
 		return errors.New("invalid format, must be source:destination")
 	}
+
+	var pos int
+	if isWindowsRegExp.MatchString(value) {
+		// We need to skip the 'C:\' that prefixes Windows directories, so we only
+		// check for ':' after that prefix.
+		pos = strings.Index(value[3:], ":") + 3
+	} else {
+		pos = strings.Index(value, ":")
```

Comment:
changing it to Index is the crux of the fix here.. we want to split on the first colon, not the last colon, because the user can supply a mount string like "/src:/dest:z" and we want to split that into "/src" and "/dest:z", not "/src:/dest" and ":z"